diff --git a/src/content/ko/user-manual/multi-agent/multi-agent-linux-installation-and-usage-guide.mdx b/src/content/ko/user-manual/multi-agent/multi-agent-linux-installation-and-usage-guide.mdx
index 7a6573e9..179be311 100644
--- a/src/content/ko/user-manual/multi-agent/multi-agent-linux-installation-and-usage-guide.mdx
+++ b/src/content/ko/user-manual/multi-agent/multi-agent-linux-installation-and-usage-guide.mdx
@@ -83,8 +83,35 @@ dpkg 명령어를 기준으로 설명합니다.
 
 1. `querypie-multi-agent.deb` 을 다운로드합니다.
 2. 다음 명령어를 실행하여 파일을 설치합니다. <br/> 출력 예시
-  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >3. 설치 여부를 확인합니다. <br/> 출력 예시
-  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >
+  ```
+sudo dpkg -i ~/Downloads/querypie-multi-agent.deb
+  ```
+  ```
+Preparing to unpack querypie-multi-agent.deb ...
+Unpacking querypie-multi-agent (3) ...
+Setting up querypie-multi-agent (3) ...
+Processing triggers for gnome-menus (3.36.0-1.1ubuntu3) ...
+Processing triggers for desktop-file-utils (0.27-2build1) ...
+Processing triggers for hicolor-icon-theme (0.17-2) ...
+  ```
+3. 설치 여부를 확인합니다. <br/> 출력 예시
+  ```
+dpkg -s querypie-multi-agent
+  ```
+  ```
+Package: querypie-multi-agent
+Status: install ok installed
+Priority: optional
+Section: devel
+Maintainer: QueryPie
+Architecture: arm64
+Version: 3
+Depends: libx11-6, libice6, libsm6, libfontconfig1
+Description: QueryPie Multi Agent
+Homepage: https://www.querypie.com/
+Copyright: QueryPie, Inc.
+  ```
+
 #### Uninstall
 
 터미널에서 다음 명령어를 실행합니다.
@@ -108,8 +135,36 @@ rpm 명령어를 기준으로 설명합니다.
 
 1. `querypie-multi-agent.rpm` 을 다운로드합니다.
 2. 터미널에서 다음 명령어를 실행하여 파일을 설치합니다. <br/> 출력 예시
-  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >3. 설치 여부를 확인합니다. <br/> 출력 예시
-  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >  <  U  n  e  x  p  e  c  t  e  d     n  o  d  e     n  a  m  e  =  a  c  :  s  t  r  u  c  t  u  r  e  d  -  m  a  c  r  o  /  >
+  ```
+sudo rpm -i ~/Downloads/querypie-multi-agent.rpm
+  ```
+  ```
+QueryPie Multi Agent has been installed.
+  ```
+3. 설치 여부를 확인합니다. <br/> 출력 예시
+  ```
+rpm -qi querypie-multi-agent
+  ```
+  ```
+Name        : querypie-multi-agent
+Version     : 3
+Release     : 202503291846
+Architecture: aarch64
+Install Date: Mon 31 Mar 2025 03:02:34 PM KST
+Group       : Applications/Development
+Size        : 493101340
+License     : Proprietary
+Signature   : (none)
+Source RPM  : querypie-multi-agent-3-202503291846.src.rpm
+Build Date  : Sun 30 Mar 2025 03:46:29 AM KST
+Build Host  : localhost
+Vendor      : QueryPie, Inc.
+URL         : https://www.querypie.com/
+Summary     : QueryPie Multi Agent
+Description :
+QueryPie Multi Agent
+  ```
+
 #### Uninstall
 
 터미널에서 다음 명령어를 실행합니다.
diff --git a/src/content/ko/user-manual/server-access-control/connecting-to-authorized-servers.mdx b/src/content/ko/user-manual/server-access-control/connecting-to-authorized-servers.mdx
index 215f310b..13a01fcd 100644
--- a/src/content/ko/user-manual/server-access-control/connecting-to-authorized-servers.mdx
+++ b/src/content/ko/user-manual/server-access-control/connecting-to-authorized-servers.mdx
@@ -60,12 +60,18 @@ A. 권한의 상태가 변경되었거나 접속이 허용되지 않은 시간
 <Callout type="info">
 **Q. RDP를 통한 Windows Server 접속은 어떻게 하나요?**
 
-A. QueryPie에서는 QueryPie User Agent를 통해서만 Windows Server에 접속이 가능합니다. 접속 방법은 아래와 같습니다.
+A. QueryPie에서는 QueryPie Agent (User Agent / Multi-Agent)를 통해서만 Windows Server에 접속이 가능합니다. 접속 방법은 아래와 같습니다.
 
-QueryPie User Agent 설치 후, 권한을 부여 받은 Windows Server를 우클릭, `Open Connection with` 메뉴를 클릭 후, PC에 설치된 RDP Client를 클릭하여 접속합니다.
+QueryPie Agent (User Agent / Multi-Agent) 설치 후, 권한을 부여 받은 Windows Server를 우클릭, `Open Connection with` 메뉴를 클릭 후, PC에 설치된 RDP Client를 클릭하여 접속합니다.
 
 * Windows PC에서는 PC에 기본 설치된 MSTSC 프로그램을 지원합니다.
 * macOS는 RDP 접속을 위해 [Windows App](https://apps.apple.com/us/app/windows-app/id1295203466?mt=12)을 다운받아야 합니다.
+
+11.3.0부터 Multi-Agent에서 Custom Account를 선택하여 계정의 암호를 수동으로 입력하여 접속할수 있도록 변경되었습니다.<br/>
+
+<figure data-layout="center" data-align="center">
+![image-20251010-095324.png](/user-manual/server-access-control/connecting-to-authorized-servers/image-20251010-095324.png)
+</figure>
 </Callout>
 
 <Callout type="info">
diff --git a/src/content/ko/user-manual/web-access-control/accessing-web-applications-websites.mdx b/src/content/ko/user-manual/web-access-control/accessing-web-applications-websites.mdx
index 33da9e34..43384a9d 100644
--- a/src/content/ko/user-manual/web-access-control/accessing-web-applications-websites.mdx
+++ b/src/content/ko/user-manual/web-access-control/accessing-web-applications-websites.mdx
@@ -17,9 +17,9 @@ QueryPie Web &gt; Web Access Control &gt; Web Apps &gt; Select a Role
 </figcaption>
 </figure>
 
-1. Web Applications 메뉴로 첫 접속시 표시되는 Role 선택 화면에서 접속을 위해 사용할 Role을 선택합니다. 
-    1. Role에 따라 웹사이트 접근 권한이 다르며, 부여된 Role이 없는 경우 Workflow로 이동합니다.
-    2. Just In Time Role를 선택한 경우, 자신이 Web Apps의 Owner 또는 Member로 지정된 Web App이 보여지며, 대상 Web App에 접속 전 Workflow의 `Web App Just-In-Time Access Request`를 통해 승인을 받고, 승인 받은 시간 동안 만 접속할 수 있습니다. 
+* Web Applications 메뉴로 첫 접속시 표시되는 Role 선택 화면에서 접속을 위해 사용할 Role을 선택합니다. 
+    * Role에 따라 웹사이트 접근 권한이 다르며, 부여된 Role이 없는 경우 Workflow로 이동합니다.
+    * Just In Time Role를 선택한 경우, 자신이 Web Apps의 Owner 또는 Member로 지정된 Web App이 보여지며, 대상 Web App에 접속 전 Workflow의 `Web App Just-In-Time Access Request`를 통해 승인을 받고, 승인 받은 시간 동안 만 접속할 수 있습니다. 
 
 ### 웹 어플리케이션 (웹 사이트)에 접속하기
 
diff --git a/src/content/ko/user-manual/workflow/requesting-db-access.mdx b/src/content/ko/user-manual/workflow/requesting-db-access.mdx
index a2d496de..fb8a6a04 100644
--- a/src/content/ko/user-manual/workflow/requesting-db-access.mdx
+++ b/src/content/ko/user-manual/workflow/requesting-db-access.mdx
@@ -49,6 +49,10 @@ import { Callout } from 'nextra/components'
 
 #### 4. 사후 승인으로 요청 보내기
 
+<Callout type="info">
+11.3.0 부터 사후 승인으로 요청을 하는 경우 결재 만료일을 지정할 수 없도록 변경되었습니다.
+</Callout>
+
 * 사후 승인 기능이 허용된 결재 규칙을 선택한 경우 Urgent Mode 스위치가 표시됩니다. 
 * Urgent mode = On 으로 설정 후 결재 요청을 등록하면 즉시 권한을 부여받거나 작업을 수행할 수 있습니다.
 
diff --git a/src/content/ko/user-manual/workflow/requesting-db-policy-exception.mdx b/src/content/ko/user-manual/workflow/requesting-db-policy-exception.mdx
index a6aa4446..cbdd68c5 100644
--- a/src/content/ko/user-manual/workflow/requesting-db-policy-exception.mdx
+++ b/src/content/ko/user-manual/workflow/requesting-db-policy-exception.mdx
@@ -71,19 +71,20 @@ A. 관리자가 Urgent Mode를 허용하지 않은 결재 규칙을 선택한 
         *  **Column :** 1개 이상 컬럼을 선택할 수 있으며, 최대 30개까지 선택할 수 있습니다.
     *  **Tag**  : 만약 신청자가 특정 경로에 정책이 태그기반으로 설정되어 있는 것을 알고 있고 어떤 태그가 부여되어 있는 지 알고 있는 경우 태그를 지정하여 신청할 수 있습니다. 일반적으로 신청자가 정책이 태그로 설정되어 있는지 어떤 태그가 사용되고 있는지 모르기 때문에 태그로 신청할 때는 먼저 관리자의 도움을 받아야 합니다. 
         *  **Tags**  : Data scope를 태그로 지정한 경우 정책에서 사용된 태그를 사용할 수 있습니다.
-*  **Data Access Expiration**  : 요청할 데이터 접근 허용 기간을 설정할 수 있습니다.
-    1. Time Duration 선택시, 분 단위로 입력 가능하고 최소는 1분, 최대는 44,640분으로 요청할 수 있습니다. 
-        1. Time Duration 의 경우, 승인 시점을 기준으로 동작합니다.
-        2. Urgent Mode 사용하는 경우, Request 등록 시점을 기준으로 데이터에 접근할 수 있습니다.
-    2. End Date 선택시, 일자와 시간을 기준으로 선택 가능하며, 최소는 현재 시간, 최대는 180일로 요청할 수 있습니다.
-*  **Reason for Request**  : 정책 예외 신청 사유를 입력합니다. 최대 1000자까지 입력할 수 있습니다.
-* 페이지 하단의 `Submit` 버튼을 눌러 요청을 상신합니다.<br/>
-  <figure data-layout="center" data-align="center">
-  ![User &gt; Workflow &gt; Submit Request &gt; DB Policy Exception Request](/user-manual/workflow/requesting-db-policy-exception/image-20250629-124654.png)
-  <figcaption>
-  User &gt; Workflow &gt; Submit Request &gt; DB Policy Exception Request
-  </figcaption>
-  </figure>
+*  **Policy Exception Schedule**  : 정책 예외가 적용되는 기간을 설정하여 요청할 수 있습니다. 최대 정책 예외의 유효 기간은 관리자가 설정한 최대 기간(Maximum Policy Exception Period)을 넘겨서 신청 수 없습니다.
+    * Specific Period : 시작 시점과 종료 시점을 지정합니다. 시간단위로 지정할 수 있습니다.
+    * Time Duration : 시작 시점으로 부터 지정한 시간까지 유효하도록 지정할 수 있습니다.
+    * Start On Approval 옵션 : Specific Period 또는 Time duration 두가지 방법 모두 Start On Approval 옵션을 활성화(체크)할 경우 승인시점 부터 효력이 발생됩니다. 일반적으로 요청하는 사람이 승인자의 승인 시점을 알 수 없기 때문에 이 옵션을 사용하는 것은 권장하지 않습니다. (승인 만료기간 제약사항이 발생되는 경우 다시 요청을 해야하는 경우가 발생될 수 있기 때문입니다.) Urgent Mode 사용하는 경우도 승인 시점은 사후이고 언제 승인이 될지 알 수 없으므로 가급적 Start On Approval 옵션은 반드시 사용해야 하는 특수한 상황을 제외하고 권장하지 않습니다.
+*  **Reason for Request**  : 정책 예외 신청 사유를 입력합니다.
+* 페이지 하단의 `Submit` 버튼을 눌러 요청을 상신합니다.
+
+<figure data-layout="center" data-align="center">
+![User &gt; Workflow &gt; Submit Request &gt; DB Policy Exception Request](/user-manual/workflow/requesting-db-policy-exception/image-20251009-144630.png)
+<figcaption>
+User &gt; Workflow &gt; Submit Request &gt; DB Policy Exception Request
+</figcaption>
+</figure>
+
 
 
 
diff --git a/src/content/ko/user-manual/workflow/requesting-restricted-data-access.mdx b/src/content/ko/user-manual/workflow/requesting-restricted-data-access.mdx
index 0ce95cd3..7ff2b8b1 100644
--- a/src/content/ko/user-manual/workflow/requesting-restricted-data-access.mdx
+++ b/src/content/ko/user-manual/workflow/requesting-restricted-data-access.mdx
@@ -62,20 +62,23 @@ A. 관리자가 Urgent Mode를 허용하지 않은 결재 규칙을 선택한 
 
 #### 5. 요청 상세정보 입력하기
 
+<Callout type="info">
+11.3.0에서 효력 만료일을 지정하는 방식이 변경되었습니다.<br/>정책예외 신청은 공통으로 “ **Policy Exception Schedule** ” 으로 항목이름이 통일되었고, 효력 시작 시점과 종료시점을 명시적으로 지정할 수 있도록 개선되었습니다.
+</Callout>
+
 *  **Title**  : 요청 제목을 입력합니다.
 *  **Connection**  : 접근 허용을 요청할 대상 테이블 또는 컬럼이 속한 커넥션을 선택합니다.
 *  **Table :**  대상 테이블을 선택합니다. 
 *  **Column :** 1개 이상 컬럼을 선택할 수 있으며, 최대 30개까지 선택할 수 있습니다.
-*  **Data Access Expiration**  : 요청할 데이터 접근 허용 기간을 설정할 수 있습니다.
-    1. Time Duration 선택시, 분 단위로 입력 가능하고 최소는 1분, 최대는 44,640분으로 요청할 수 있습니다. 
-        1. Time Duration 의 경우, 승인 시점을 기준으로 동작합니다.
-        2. Urgent Mode 사용하는 경우, Request 등록 시점을 기준으로 데이터에 접근할 수 있습니다.
-    2. End Date 선택시, 일자와 시간을 기준으로 선택 가능하며, 최소는 현재 시간, 최대는 30일로 요청할 수 있습니다.
-*  **Comment**  : 접근 허용 요청 사유를 입력합니다.
+*  **Policy Exception Schedule**  : Data Access 정책 예외가 적용되는 기간을 설정하여 요청할 수 있습니다. 최대 정책 예외의 유효 기간은 관리자가 설정한 최대 기간(Maximum Policy Exception Period)을 넘겨서 신청 수 없습니다.
+    * Specific Period : 시작 시점과 종료 시점을 지정합니다. 시간단위로 지정할 수 있습니다.
+    * Time Duration : 시작 시점으로 부터 지정한 시간까지 유효하도록 지정할 수 있습니다.
+    * Start On Approval 옵션 : Specific Period 또는 Time duration 두가지 방법 모두 Start On Approval 옵션을 활성화(체크)할 경우 승인시점 부터 효력이 발생됩니다. 일반적으로 요청하는 사람이 승인자의 승인 시점을 알 수 없기 때문에 이 옵션을 사용하는 것은 권장하지 않습니다. (승인 만료기간 제약사항이 발생되는 경우 다시 요청을 해야하는 경우가 발생될 수 있기 때문입니다.) Urgent Mode 사용하는 경우도 승인 시점은 사후이고 언제 승인이 될지 알 수 없으므로 가급적 Start On Approval 옵션은 반드시 사용해야 하는 특수한 상황을 제외하고 권장하지 않습니다.
+*  **Reason for Request**  : 접근 허용 요청 사유를 입력합니다.
 * 페이지 하단의 `Submit` 버튼을 눌러 요청을 상신합니다.
 
 <figure data-layout="center" data-align="center">
-![User &gt; Workflow &gt; Submit Request &gt; Restricted Data Access Request &gt; Request Summary](/user-manual/workflow/requesting-restricted-data-access/image-20250626-225723.png)
+![User &gt; Workflow &gt; Submit Request &gt; Restricted Data Access Request &gt; Request Summary](/user-manual/workflow/requesting-restricted-data-access/image-20251009-140741.png)
 <figcaption>
 User &gt; Workflow &gt; Submit Request &gt; Restricted Data Access Request &gt; Request Summary
 </figcaption>
@@ -83,3 +86,5 @@ User &gt; Workflow &gt; Submit Request &gt; Restricted Data Access Request &gt;
 
 
 
+
+
diff --git a/src/content/ko/user-manual/workflow/requesting-unmasking-mask-removal-request.mdx b/src/content/ko/user-manual/workflow/requesting-unmasking-mask-removal-request.mdx
index d3c647ff..7e2b9671 100644
--- a/src/content/ko/user-manual/workflow/requesting-unmasking-mask-removal-request.mdx
+++ b/src/content/ko/user-manual/workflow/requesting-unmasking-mask-removal-request.mdx
@@ -62,19 +62,22 @@ A. 관리자가 Urgent Mode를 허용하지 않은 결재 규칙을 선택한 
 
 #### 5. 요청 상세정보 입력하기
 
+<Callout type="info">
+11.3.0에서 효력 만료일을 지정하는 방식이 변경되었습니다.<br/>정책예외 신청은 공통으로 “ **Policy Exception Schedule** ” 으로 항목이름이 통일되었고, 효력 시작 시점과 종료시점을 명시적으로 지정할 수 있도록 개선되었습니다.
+</Callout>
+
 *  **Title**  : 요청 제목을 입력합니다.
 *  **Connection**  : 마스킹 해제를 요청할 대상 컬럼이 속한 커넥션을 선택합니다.
 *  **Column :** 1개 이상 컬럼을 선택할 수 있으며, 최대 30개까지 선택할 수 있습니다.
-*  **Unmasking Expiration**  : 마스킹 해제를 요청할 기간을 설정할 수 있습니다.
-    1. Time Duration 선택시, 분 단위로 입력 가능하고 최소는 1분, 최대는 44,640분으로 요청할 수 있습니다. 
-        1. Time Duration 의 경우, 승인 시점을 기준으로 동작합니다.
-        2. Urgent Mode 사용하는 경우, Request 등록 시점을 기준으로 마스킹 해제된 데이터를 확인할 수 있습니다.
-    2. End Date 선택시, 일자와 시간을 기준으로 선택 가능하며, 최소는 현재 시간, 최대는 30일로 요청할 수 있습니다.
-*  **Comment**  : 마스킹 해제 요청 사유를 입력합니다.
+*  **Policy Exception Schedule**  : 마스킹 해제가 적용되는 기간을 설정하여 요청할 수 있습니다. 최대 정책 예외의 유효 기간은 관리자가 설정한 최대 기간(Maximum Policy Exception Period)을 넘겨서 신청 수 없습니다.
+    * Specific Period : 시작 시점과 종료 시점을 지정합니다. 시간단위로 지정할 수 있습니다. 
+    * Time Duration : 시작 시점으로 부터 지정한 시간까지 유효하도록 지정할 수 있습니다. 
+    * Start On Approval 옵션 : Specific Period 또는 Time duration 두가지 방법 모두 Start On Approval 옵션을 활성화(체크)할 경우 승인시점 부터 효력이 발생됩니다. 일반적으로 요청하는 사람이 승인자의 승인 시점을 알 수 없기 때문에 이 옵션을 사용하는 것은 권장하지 않습니다. (승인 만료기간 제약사항이 발생되는 경우 다시 요청을 해야하는 경우가 발생될 수 있기 때문입니다.) Urgent Mode 사용하는 경우도 승인 시점은 사후이고 언제 승인이 될지 알 수 없으므로 가급적 Start On Approval 옵션은 반드시 사용해야 하는 특수한 상황을 제외하고 권장하지 않습니다.
+*  **Reason for Request**  : 마스킹 해제 요청 사유를 입력합니다.
 * 페이지 하단의 `Submit` 버튼을 눌러 요청을 상신합니다.
 
 <figure data-layout="center" data-align="center">
-![User &gt; Workflow &gt; Submit Request &gt; Unmasking Request &gt; Request Summary](/user-manual/workflow/requesting-unmasking-mask-removal-request/image-20241103-071004.png)
+![User &gt; Workflow &gt; Submit Request &gt; Unmasking Request &gt; Request Summary](/user-manual/workflow/requesting-unmasking-mask-removal-request/image-20251009-135940.png)
 <figcaption>
 User &gt; Workflow &gt; Submit Request &gt; Unmasking Request &gt; Request Summary
 </figcaption>
