--- a/original.mdx
+++ b/improved.mdx
@@ -142,8 +142,8 @@
 </figcaption>
 </figure>
 
-* 이후 해당 서버에 접속 가능한 계정이 여러개라면, Account 선택창이 열립니다. 
-* 사용하려는 계정을 선택하고 필요시 비밀번호를 입력한 뒤, `OK` 버튼을 클릭하여 세션을 엽니다.
+* 이후 해당 서버에 접속 가능한 계정이 여러 개라면, Account 선택창이 열립니다. 
+* 사용하려는 계정을 선택하고 필요 시 비밀번호를 입력한 뒤, `OK` 버튼을 클릭하여 세션을 엽니다.
 
 <figure data-layout="center" data-align="center">
 <img src="/user-manual/user-agent/screenshot-20240730-162532.png" alt="Agent &gt; Server &gt; Open New Session" width="746" />
