--- a/original.mdx
+++ b/improved.mdx
@@ -91,9 +91,9 @@
 ### Attribute 기반 승인자 지정 기능
 
 <Callout type="info">
-11.4.0부터 속성 기반 승인자 지정시 여러개의 속성을 지정할 수 있도록 개선되었습니다.
+11.4.0부터 속성 기반 승인자 지정 시 여러 개의 속성을 지정할 수 있도록 개선되었습니다.
 * Admin &gt; General &gt; Company Management &gt; General &gt; Workflow Configurations의 Use User Attribute-Based Approval 옵션 스위치가 제거되었습니다.
-* 기존 Attribute 기반 승인자 지정시 하나의 Attribute만 지정할 수 있었으나 다른 Attribute를 선택할 수 있도록 하여 각 승인 단계별 각각 다른 속성의 승인자가 매핑되도록 개선되었습니다. 
+* 기존 Attribute 기반 승인자 지정 시 하나의 Attribute만 지정할 수 있었으나 다른 Attribute를 선택할 수 있도록 하여 각 승인 단계별 각각 다른 속성의 승인자가 매핑되도록 개선되었습니다. 
 </Callout>
 
 사용자 프로필의 특정 Attribute(예: 팀 리더, 부서장)를 기준으로 승인자를 자동으로 지정할 수 있습니다.
@@ -119,7 +119,7 @@
           </figure>
 * Attribute 값은 있지만 해당 사용자가 비활성화된 경우
     * 만약 상신자 User Profile에 해당 Attribute 값으로 지정된 승인자가 비활성화된 경우, 워크플로우 상신 시 Approver란에 지정된 승인자가 표기되지 않으며 아래와 같은 내용의 알림 모달이 표시되며 요청 제출이 중단됩니다.
-        * 에러 메세지 예시:
+        * 에러 메시지 예시:
           <figure data-layout="center" data-align="center">
           <img src="/administrator-manual/general/company-management/general/Screenshot-2025-06-16-at-10.30.14-AM.png" alt="Screenshot-2025-06-16-at-10.30.14-AM.png" width="615" />
           </figure>
@@ -164,14 +164,14 @@
 
 ### SQL 요청 실행 시 쓰로틀링 활성화 (Use throttling when submitting SQL Request in Workflow)
 
-토글을 켜면 Workflow에서 상신된 SQL 요청 실행시 쓰로틀링을 적용합니다.
+토글을 켜면 Workflow에서 상신된 SQL 요청 실행 시 쓰로틀링을 적용합니다.
 
 <figure data-layout="center" data-align="center">
 <img src="/administrator-manual/general/company-management/general/Screenshot-2025-05-09-at-3.18.32-PM.png" alt="Screenshot-2025-05-09-at-3.18.32-PM.png" width="604" />
 </figure>
 
 
-이 옵션을 활성화하는 경우, 명시된 갯수만큼 쿼리를 연속적으로 실행한 뒤, 명시된 간격(ms)만큼 대기합니다.
+이 옵션을 활성화하는 경우, 명시된 개수만큼 쿼리를 연속적으로 실행한 뒤, 명시된 간격(ms)만큼 대기합니다.
 이를 통해 대용량 쿼리 실행에 따르는 DB 부하를 방지할 수 있습니다.
 
 *  **Query chunk size**  : 1~999,999 사이의 값으로 설정되며, 다수의 쿼리를 해당 개수만큼 나누어 순차적으로 실행하는 기능
