--- a/original.mdx
+++ b/improved.mdx
@@ -839,7 +839,7 @@
     1. 필터값: Active, Locked, Expired, Locked Manually
 4.  **Time Range**  : DB Access History 또는 Server Access History 선택 시 시간 범위로 필터링합니다.
     1. 시작 시간 및 종료 시간을 입력합니다.
-    2. 입력된 시간 범위 내의 결과를 제외할 지 여부를 선택합니다. 선택 시, 입력된 시간 범위 외의 로그만 출력됩니다.
+    2. 입력된 시간 범위 내의 결과를 제외할지 여부를 선택합니다. 선택 시, 입력된 시간 범위 외의 로그만 출력됩니다.
 5.  **Result**  : Server Access History 선택 시 결과를 기준으로 필터링합니다.
     1. 필터값: Success, Failure
 6.  **Action Type**  : Server Access History 선택 시 행위 유형(Action Type)을 기준으로 필터링합니다.
