--- a/original.mdx
+++ b/improved.mdx
@@ -48,7 +48,7 @@
     * Privileges는 "-"로 표시됩니다.
 3.  **SQL Tool에서 접속** 
     * DataGrip이나 기타 SQL Tool을 실행합니다.
-    *  **Host와 Port는 Connection Guide에 명시되어있는 정보를 입력해야합니다.**  
+    *  **Host와 Port는 Connection Guide에 명시되어있는 정보를 입력해야 합니다.**  
     * 벤더에 따라 추가 정보를 입력합니다.
       <figure data-layout="center" data-align="center">
       <img src="/user-manual/database-access-control/connecting-to-custom-data-source/Screenshot-2025-03-06-at-2.18.26-PM.png" alt="Screenshot-2025-03-06-at-2.18.26-PM.png" width="712" />
